addons/textalog/assets/themes/default_theme/plain_black.tres

```diff
@@ -86,5 +86,10 @@ Button/styles/hover = SubResource( 3 )
 Button/styles/normal = SubResource( 4 )
 Button/styles/pressed = SubResource( 5 )
 DialogNode/styles/bubble = null
+<<<<<<< HEAD
```

Comment:
Diff commit header. This should be solved before adding more commits

Reply:
aw god damnit, thought I resolved that